assets/js/modules/analytics/datastore/properties.test.js

```diff
@@ -38,6 +41,8 @@ describe( 'modules/analytics properties', () => {
 	} );
 
 	beforeEach( () => {
+		enabledFeatures.add( 'ga4setup' );
```

Comment:
This is only needed for tests in the `getPropertiesIncludingGA4` group. Let's define a new `beforeEach` in that group and enable the `ga4setup` feature there.